HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -5117,24 +5117,29 @@ def self.calculate_heat_pump_defrost_load_power_watts(heat_pump, unit_multiplier
     power_design = fan_watts_per_cfm * design_airflow_1x
```

Comment:
All these `foo_1x` variables are pretty confusing. It'd be simpler if you did the calculations normally (without adjustments for the unit multiplier) and then just apply the unit multiplier as needed at the point at which you are assigning values to the EnergyPlus objects. For example, you might apply the unit multiplier to an object field [like this](https://github.com/NREL/OpenStudio-HPXML/blob/88e256c9f0be00a35eca1d521513d644af8631fe/HPXMLtoOpenStudio/resources/waterheater.rb#L953) or apply the unit multiplier to an EMS actuator [like this](https://github.com/NREL/OpenStudio-HPXML/blob/88e256c9f0be00a35eca1d521513d644af8631fe/HPXMLtoOpenStudio/resources/waterheater.rb#L1782-L1784).

Reply:
I think the challenge here is that the airflow and capacities are already with unit multiplier at calling, but the regression is based on capacity per system, so I have to revert first then apply unit multiplier at the end. I've tried my best to simplify a bit and added a comment here.

Reply:
Ah, fair enough, that's a good reason to keep it how it is. Thanks for doing what you could do to simplify it.